testData/solutions/win32.json

```diff
@@ -9,14 +9,248 @@
+                    "Braille\\.BrailleMode": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInputPath": "http://registry\\.gpii\\.net/common/brailleMode",
+                            "match": {
+                                "line": 0,
+                                "structured": 1,
+                                "speechHistory": 2
+                            }
+                        }
+                    },
+                    "options\\.SayAllIgnoreShiftKeys": "http://registry\\.gpii\\.net/common/stickyKeys"
+                }
+            },
+            "configuration2": {
                 "type": "gpii.settingsHandlers.INISettingsHandler",
                 "options": {
-                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\15.0\\Settings\\VoiceProfiles.ini"
+                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\17.0\\Settings\\VoiceProfiles\\GPII.VPF"
+                },
+                "supportedSettings": {
+                    "Options.PrimarySynthesizer": {},
+                    "ENU-Global.Rate": {},
+                    "ENU-JAWSCursor.Rate": {},
+                    "ENU-Keyboard.Rate": {},
+                    "ENU-MenuAndDialog.Rate": {},
+                    "ENU-Message.Rate": {},
+                    "ENU-PCCursor.Rate": {},
+                    "ENU-Global.Pitch": {},
+                    "ENU-JAWSCursor.Pitch": {},
+                    "ENU-Keyboard.Pitch": {},
+                    "ENU-MenuAndDialog.Pitch": {},
+                    "ENU-Message.Pitch": {},
+                    "ENU-PCCursor.Pitch": {},
+                    "ENU-Global.Volume": {},
+                    "ENU-JAWSCursor.Volume": {},
+                    "ENU-Keyboard.Volume": {},
+                    "ENU-MenuAndDialog.Volume": {},
+                    "ENU-Message.Volume": {},
+                    "ENU-PCCursor.Volume": {},
+                    "ENU-Global.Punctuation": {},
+                    "ENU-JAWSCursor.Punctuation": {},
+                    "ENU-Keyboard.Punctuation": {},
+                    "ENU-MenuAndDialog.Punctuation": {},
+                    "ENU-Message.Punctuation": {},
+                    "ENU-PCCursor.Punctuation": {},
+                    "ENU-Global.SynthLangString": {},
+                    "ENU-JAWSCursor.SynthLangString": {},
+                    "ENU-Keyboard.SynthLangString": {},
+                    "ENU-MenuAndDialog.SynthLangString": {},
+                    "ENU-Message.SynthLangString": {},
+                    "ENU-PCCursor.SynthLangString": {}
                 },
                 "capabilities": [],
                 "capabilitiesTransformations": {
-                    "cloud4allVoiceProfile-GlobalContext\\.Punctuation": {
+                    "Options\\.PrimarySynthesizer": "http://registry\\.gpii\\.net/common/speechSynthesizer",
```

Comment:
"speechSynthetiser" is not a known common term. Are there plans to include it? If it's not the case then I guess that this transformation must be removed?